### DISCUSSION TITLE
Taking 41s for plausible.js to load

### DISCUSSION THREAD

**Discussion by craigschmidt** [plausible/analytics#1620] [Anything Else]
Hi There,

I'm seeing very long load times of for `plausible.js` on my site `www.delomore.com`. A friend of mine saw ~40 seconds when he used my site. It is a SvelteKit site hosted on Vercel. Is anyone else seeing long load times?  Have I configured something incorrectly?

It seems like it is 4-6s on my computer.

![Screen Shot 2022-01-19 at 12 51 45 PM](https://user-images.githubusercontent.com/513771/150186744-944852ea-24f7-47b0-8e0d-855c1c0de8d4.png)

**Comment by craigschmidt** [comment]
There you go, I got 43 seconds this last time. 

![Screen Shot 2022-01-19 at 1 00 19 PM](https://user-images.githubusercontent.com/513771/150187727-438961f5-6cf1-4325-92da-5ffd9f7cf89f.png)

**Comment by metmarkosaric** [comment]
thanks for reporting that @craigschmidt! we're using a global CDN Bunny for loading the script around the world. it loads fast for me from western Europe. where are you based?

**Comment by craigschmidt** [comment]
I’m in Boston, Massachusetts.  My friend who also saw slow times is in Ottawa, Canada.  We both see slow times quite consistently.

-Craig

**Reply by metmarkosaric** [reply]
we've had couple more reports from the east coast in the US in the last hour so seems like Bunny CDN has some issues in that area. we've contacted them and will follow up as soon as we hear back. sorry for the inconvenience!

**Comment by metmarkosaric** [comment]
could you try again now please?

**Comment by craigschmidt** [comment]
Hey, it is 21ms now.  Thanks for following up with the Bunny CDN folks.

**Reply by metmarkosaric** [reply]
great, thanks for the update! we've also made some tweaks to the way our script is being served globally so this should not happen again.
